pkgs/tools/misc/sanctity/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, stdenv, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sanctity";
+  version = "0.7.3";
+
+  src = fetchGit {
+    url = "https://codeberg.org/papojari/sanctity.git";
+    ref = "refs/tags/v0.7.3";
+    rev = "5af70fca350b24b03ad8e0eadc48eb0afd96cc5a";
+  };
```

```suggestion
  src = fetchFromGitea {
    domain = "codeberg.org";
    owner = "papojari";
    repo = pname;
    rev = "v${version}";
    sha256 = "sha256-1bgk4HxWdefhNFeEQrg9Jvj3aMvzFfIJl+eYK/AgX5g=";
  };
```
The detail is https://github.com/NixOS/nixpkgs/pull/162007#discussion_r815430691